packages/shared/src/components/post/write/WriteLinkPreview.tsx

```diff
@@ -48,7 +48,7 @@ export function WriteLinkPreview({
         <WritePreviewContent className={isMinimized && '!py-2 !px-3'}>
           <div className="flex flex-col flex-1 typo-footnote">
             <span className="font-bold line-clamp-2">{preview.title}</span>
-            {preview.source &&
+            {preview.source?.id !== 'unknown' &&
```

Comment:
This does not feel right, but based on my understanding of how these are stored in the db, it's the only way to determine if this is a URL preview of an external page, or a preview of an internal post

Reply:
Yep it's a set constraint at the moment :)